core/transport/src/main/proto/grpc/meta_master.proto

```diff
@@ -264,6 +272,11 @@ service MetaMasterMasterService {
    */
   rpc GetMasterId(GetMasterIdPRequest) returns (GetMasterIdPResponse);
 
+  /**
+ * Returns a master id for the given master address.
+ */
+  rpc GetClusterIdInternal(GetClusterIdInternalPRequest) returns (GetClusterIdInternalPResponse);
```

Comment:
If the server-side is running in old version, and the worker use the version of yours which contains this PR, what will happen? Please list the consistent issue if we upgrade a part of master or worker cluster.

Reply:
Does Alluxio have requirements for the health status of the cluster during the upgrade process? 
I may need to add compatibility design to the upgrade process